sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-part-time-msfaa-columns.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN part_time_msfaa_number CHAR(10),
```

Comment:
The reason behind `part_time_msfaa_number` saved as char is just because the msfaa_number in individuals is following the same? I was wondering why should not use the INT in DB as we use the same data type in sims.msfaa_numbers ?

Reply:
MSFAA for full-time is CHAR(10) hence following the same pattern already in place for the same table.